content/en/tracing/trace_collection/custom_instrumentation/go/migration.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Migrating from v1 to v2 of the Go Tracer
+description: 'Upgrade your Go tracer from v1 to v2.'
+further_reading:
+    - link: 'tracing/trace_collection/custom_instrumentation/dd-api'
+      tag: 'Documentation'
+      text: 'Get started with v1 of the Go Tracer'
+    - link: 'tracing/trace_collection/custom_instrumentation/dd-api-v2'
+      tag: 'Documentation'
+      text: 'Get started with v2 of the Go Tracer'
+---
+
+<div class="alert alert-info">This documentation assumes that you are currently using version v1.x of the Go Tracer. If you are already on v2.x of the tracer, please visit <a href="tracing/trace_collection/custom_instrumentation/dd-api-v2">Go v2 Custom Instrumentation using the Datadog API</a> documentation instead.</div>
+
+This document outlines migrating from an older version of the Datadog tracer (v1.x.x) to v2.
+
+Datadog's v2 version of the Go tracer provides a huge refactor of our API, moving away from interfaces to provide flexibility in future works, isolating our integrations to prevent false-positives from security scanners, and enforcing proper library patterns to prevent misuse. This update is the result of continuous feedback from customers, the community, as well as our extensive internal usage. 
+
+To assist with the migration from an older version of the tracer, we have also provided a new migration tool to help with the most essential changes made in v2.
```

Comment:
```suggestion
Version 2 of the Go tracer introduces significant API improvements:

- Removes interfaces to enable future flexibility
- Isolates integrations to prevent false positives from security scanners
- Enforces library patterns to prevent misuse

To simplify the migration process, Datadog provides a migration tool that handles essential code updates automatically.
```
```suggestion
Datadog's v2 version of the Go tracer provides a huge refactor of our API, moving away from interfaces to provide flexibility in future works, isolating our integrations to prevent false-positives from security scanners, and enforcing proper library patterns to prevent misuse. This update is the result of continuous feedback from customers, the community, as well as our extensive internal usage. 

To assist with the migration from an older version of the tracer, we have also provided a new migration tool to help with the most essential changes made in v2.
```